pyproject.toml

```diff
@@ -79,9 +79,14 @@ test-command = "pytest --pyargs shapely.tests"
 source = ["shapely"]
 omit = ["shapely/tests/*"]
 
+[tool.doc8]
+include = ["docs"]
+ignore = ["D000", "D004"]
+allow-long-titles = true
+
 [tool.ruff]
 line-length = 88
-extend-exclude = ["docs/*", "benchmarks/*", "shapely/_version.py"]
+extend-exclude = ["docs/*", "benchmarks/*", "shapely/_version.py", "versioneer.py"]
```

Comment:
I've taken this suggested edit for #2141 (also removing `files: 'shapely/'` from the pre-commit config).